deploy/helm/scf/assets/operations/instance_groups/diego-cell.yaml

```diff
@@ -42,30 +41,88 @@
 - type: replace
   path: /instance_groups/name=diego-cell/jobs/name=cfdot/properties/locket?/hostname
   value: ((deployment-name))-diego-api
-
-# Add quarks properties.
 - type: replace
-  path: /instance_groups/name=diego-cell/jobs/name=garden/properties/quarks?
-  value:
-    run:
-      memory: 4096
-      virtual-cpus: 2
-      # healthcheck:
-      #   garden:
-      #     readiness: &probe
-      #       exec:
-      #         command: ['sh', '-c', 'ss -nlt | grep "LISTEN.*:17019"']
-      #     liveness: *probe
+  path: /instance_groups/name=diego-cell/jobs/name=rep/properties/set_kernel_parameters?
+  value: false
 
 # Temporarily remove garden network_plugin.
 - type: remove
   path: /instance_groups/name=diego-cell/jobs/name=garden/properties/garden/network_plugin
 - type: remove
   path: /instance_groups/name=diego-cell/jobs/name=garden/properties/garden/network_plugin_extra_args
 
+# Add bosh_containerization properties for the rep.
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name=rep/properties/bosh_containerization?
+  value:
+    ports:
+    - name: rep
+      protocol: TCP
+      internal: 1800
+    - name: rep-tls
+      protocol: TCP
+      internal: 1801
+
 - type: replace
   path: /instance_groups/name=diego-cell/jobs/name=cflinuxfs3-rootfs-setup/properties/quarks?/bpm/processes
   value: []
 - type: replace
   path: /instance_groups/name=diego-cell/jobs/name=cfdot/properties/quarks?/bpm/processes
   value: []
+
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name=rep/properties?/diego?/rep?/locket?/api_location?
+  value: ((deployment-name))-diego-api:8891
+
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name=rep/properties?/diego?/rep?/bbs?/api_location?
+  value: ((deployment-name))-diego-api:8889
+
+# TODO: the namespace here has to be determined automatically - helm can help here?
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name=rep/properties?/diego?/rep?/advertise_domain?
+  value: scf.svc.cluster.local
+
+# TODO: this should actually be the container's hostname
```

Comment:
Should we patch the usages of this property to perform a call to `hostname`?

Reply:
Every usage of `cell_id` is being rendered to `scf-diego-cell-0` (scf being the deployment name). The hostname of the container, on the other hand, is `scf-diego-cell-v1-0`.